KomaMRICore/src/KomaMRICore.jl

```diff
@@ -13,7 +13,8 @@ using Interpolations
 using CUDA
 #Reconstruction
 using MRIBase
-@reexport using MRIBase: EncodingCounters, Profile, RawAcquisitionData, AcquisitionData, AcquisitionHeader
+import MRIBase: EncodingCounters
```

Comment:
??

Reply:
You are right. Removed in https://github.com/cncastillo/KomaMRI.jl/pull/231/commits/219412116a64d553b00080f06b69797717de81c2